x/concentrated-liquidity/swaps_test.go

```diff
@@ -3017,7 +3043,7 @@ func (s *KeeperTestSuite) TestFunctionalSwaps() {
 	expectedSqrtPrice := osmomath.MustNewDecFromStr("71.74138432587113364823838192")
 	actualSqrtPrice := osmomath.BigDecFromSDKDec(clPool.GetCurrentSqrtPrice())
 	expectedTokenIn := swapCoin1.Amount.Mul(sdk.NewInt(int64(positions.numSwaps)))
-	expectedTokenOut := sdk.NewInt(983645)
+	expectedTokenOut := sdk.NewInt(982676)
```

Comment:
All of these new numbers are the correct numbers that show in the python code in the comments. Someone changed them in order to use .02% fee but I changed this back to .03% fee like the python code shows